test/functionality1/moderation.sh

```diff
@@ -174,6 +174,19 @@ specific_photo_is_banned () {
   [[ "$(q "select count(*) from photo")" -eq 2 ]]
 }
 
+# When we ban an account, their normalized_email should be added to banned_person
+normalized_added_to_banned () {
+  setup 0 true
+
+  assume_role reporter
+
+  jc POST "/skip/${accused_id}" -d '{ "report_reason": "n/a" }'
+
+  c GET "/admin/ban/$(ban_token)"
+
+  [[ "$(q "select count(*) from banned_person where email = 'accused@gmail.com'")" -eq 1 ]]
```

Comment:
I don't understand why `accused@gmail.com` gets added to `banned_person` when `accused+1@example.com` is who signs up

Reply:
because the normalize_email function simply adds @gmail.com to the end since it only checks for gmail and googlemail, I can change it if that'd be better

Reply:
But if I'm reading the test properly, `accused+1` isn't at `gmail.com`, is he? He's at `example.com`. I think I'm misreading, because the logic being tested looks correct

Reply:
Yes but I added example.com to also be normalized, to make the tests work properly https://github.com/duolicious/duolicious-backend/blob/43e933c3ef40486cce703666e43025ebfb6d37d8/service/person/__init__.py#L276

Reply:
Ah! I missed that part! Sorry. I think that's a good idea!

Unrelated: now that I'm looking at this line, could it be changed to this?

```python
if domain in ["gmail.com", "googlemail.com", "example.com"]:
```